package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@quero/guidelines",
+  "version": "1.0.0",
+  "description": "The definitive code-style guide by Quero Educação",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "lint": "eslint --ext .js --fix --no-eslintrc -c .eslintrc.js ./packages/eslint-config-typescript",
+    "pretest": "npm run lint",
+    "test": "jest",
+    "prerelease": "npm run lint",
+    "release": "lerna version",
```

Comment:
por hora eu não habilitei a opção criar uma release no github  pq pra isso, a pessoa que for mandar um release precisa ter um token do github disponivel no seu shell pra mandar aqui se não a release quebra.